doc/index.rst

```diff
@@ -33,7 +33,7 @@ The following sensors are supported:
 
 .. toctree::
    campbellscientific_CSAT3BH_sensor
-   gil_windsonic_2-d_sonic_wind_sensor
+   gill_windsonic_2-d_sonic_wind_sensor
```

Comment:
If this is a key in a dictionary I gently suggest you shorten it a bit, if that is easy to d. Even just changing 2-d to 2d would reduce punctuation uncertainty (since no hyphens appear anywhere else and users might wonder if it's 2_d or 2-d).

Reply:
A good suggestion but it is not a key in a dictionary (otherwise also code would have had to be corrected) so I will leave it as is.